ci/build/npm-postinstall.sh

```diff
@@ -56,18 +56,21 @@ main() {
   fi
 }
 
+# This is a copy of symlink_asar in ../lib.sh. Look there for details.
```

Comment:
can we import that lib and share code instead of duplicating it? we have a general problem (especially in our other projects) that we have unnecessary copies of things, and that often results in inconsistencies between those files over time

Reply:
Yeah that makes a lot of sense, in fact they were already out of sync with this version using `-e` and the other using `-L`. I opened https://github.com/cdr/code-server/issues/3371